layouts/partials/head.html

```diff
@@ -3,6 +3,11 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
     <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0">
 
+    <!-- If the site is not running in production let's block search engines from indexing. -->
+    {{ if ne hugo.Environment "production" }}
+        <meta name="robots" content="noindex">
+    {{ end }}
```

Comment:
We actually do this already in robots.txt (driven mainly by all the PR previews we do):

https://github.com/pulumi/docs/blob/98ae287732c3a17de86455228a9bcde641b02858/layouts/robots.txt#L1-L4

```suggestion
```

Reply:
...although now I see that neither of these approaches seems to be working with dev stacks (though previews are good) because we seem to be running `production` builds for them. I'll get an issue filed for fixing this in the build (and add a bit more documentation for creating and tearing down dev stacks).

Reply:
Ah, thanks for pointing this out.